mappings/net/minecraft/util/math/AreaHelper.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_2424 net/minecraft/AreaHelper
+CLASS net/minecraft/class_2424 net/minecraft/util/math/AreaHelper
```

Comment:
Not limited to math since it interacts with the world. It was originally an inner of like the portal block, so I'd expect this to be in like dimension package or similar.